settings.php

```diff
@@ -0,0 +1,62 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Settings for the Moodle Multiblock.
+ *
+ * @package   block_multiblock
+ * @copyright 2021 Muhammad Ali <ma2716@bath.ac.uk>
+ * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+defined('MOODLE_INTERNAL') || die;
+
+if ($ADMIN->fulltree) {
+    global $DB, $PAGE, $CFG;
+
+    $blocks = $DB->get_records('block', array(), 'name ASC');
+
+    // Multiblock title (heading).
+    $settings->add(new admin_setting_configtext('block_multiblock/title', get_string('moodle_multiblock_title', 'block_multiblock'),
+    get_string('moodle_multiblock_title_desc', 'block_multiblock'), get_string('pluginname_recommended', 'block_multiblock'),
```

Comment:
I have removed this code for handling the block title so the language strings are the way to now define the block title.

Reply:
Hi James,
Sorry it might not have been clear what I meant, I was specifically talking about the use of the 'pluginname_reccommended' string to set a default title if none is set. The name "Course Toolkit" is specific to Bath's use case so it doesn't make sense for this to be the fallback for everyone. Setting this via a lang string instead works, but I'm happy to keep the setting here for the additional flexibility it provides, just have it default to an empty string, and have the block use "Multiblock" as the title if the setting is empty.